invenio_accounts/ext.py

```diff
@@ -144,6 +147,17 @@ def init_app(self, app, sessionstore=None, register_blueprint=True):
         # Monkey-patch Flask-Security
         InvenioAccounts.monkey_patch_flask_security()
 
+        # Override Flask-Security's default login view function
+        @app.before_first_request
+        def override_login_view():
+            # this is only done before the first request, to give other
```

Comment:
I read this comment and it makes sense.
I am just wondering why it cannot be done simply when creating this extension and not in the `before_first_request`.
Is it because otherwise we cannot be sure that the `security` extension is already loaded (issue with the extensions loading order?)

Reply:
brief testing would always give me the view function configured in `invenio_accounts.config` if not delayed until `before_first_request`
but this may have been due to another issue (the `invenio_oauthclient` not overriding the default config value); gonna investigate!

Reply:
indeed, it works after resolving that other issue, resolved!